config/environments/development.rb

```diff
@@ -45,14 +48,6 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
-
-  config.hosts += [
-    "authenticating-proxy.dev.gov.uk",
-  ]
```

Comment:
I believe we'll want to keep the hosts config as this app does run in Docker and connecting via the host is mentioned in the README: https://github.com/alphagov/authenticating-proxy/blob/main/README.md

Reply:
Yes, good point. I'll revert.

Reply:
Fixed.